src/workloads/execution/BestBuyInserts.yml

```diff
@@ -0,0 +1,141 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload measures bulk insert performance with the BestBuy dataset used in some other
```

Comment:
rather than vaguely mentioning "some other workloads" I'd provide here the link to the dataset. I understand that the link can be found in one of the other two repos that you had to touch to make this workload, but... you know, the future readers would appreciate having it here as well :)